### ISSUE TITLE
on clicking shift and scrolling the scroll doesn't work

### ISSUE THREAD

**Issue by ranjan-ranjan** [josdejong/jsoneditor#437]

**Comment by josdejong** [comment]
I suppose you mean holding the keyboard shift key down? There is no need to hold shift down whilst scrolling, nor is there any function attached. I'm not sure what the problem is, can you elaborate?

**Comment by ranjan-ranjan** [comment]
yes, thats the problem. it is required when i want to select.same as in Word.

**Comment by josdejong** [comment]
I'm not familiar with this "select same" feature of Word, but I understand it has something to do with selecting all occurrences of a selected word or so? 

JSONEditor doesn't support multi selection sort of things. If there is a good use case we can consider it.

**Comment by ranjan-ranjan** [comment]
yes

**Comment by josdejong** [comment]
Ok then, I've changed the title of this issue to reflect the feature we're looking for. I don't expect to implement this any time soon (at least not myself), but lets see if there are more people interested in this feature.

**Comment by rainAgain** [comment]
I need the feature :smiley:
